### PR TITLE
MAR-4608:  Upgrade to Python 3

### PR DESCRIPTION
**Pull request by Reshma-M-A**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: python
 python:
   - "2.7"
```
Comment (Reshma-M-A) [inline .travis.yml:3]:
```suggestion
```
Should we remove this since we no longer need it anymore?

File: app/main/helpers/diff_tools.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import difflib
 try:
-    from itertools import izip_longest
+    from itertools import zip_longest
 except ImportError:
     from itertools import zip_longest as izip_longest
```
Comment (Reshma-M-A) [inline app/main/helpers/diff_tools.py:5]:
```suggestion
```
Should we remove the `ImportError`? or at least change `izip_longest`?

File: tests/app/main/views/test_users.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # coding=utf-8
```
Comment (Reshma-M-A) [inline tests/app/main/views/test_users.py:1]:
```suggestion
```
Not sure if this is a relevant comment to have.

File: tests/app/main/views/test_users.py [hunk 4]
```diff
@@ -245,7 +245,7 @@ def _assert_things_about_user_export(self, response, users):
             assert sorted(list(users[0].keys())) == sorted(rows[0])
 
             for index, user in enumerate(users):
-                assert sorted([six.text_type(val) for val in user.values()]) == sorted(rows[index+1])
+                assert sorted([six.text_type(val) for val in list(user.values())]) == sorted(rows[index+1])
 
     ##########################################################################
```
Comment (Reshma-M-A) [inline tests/app/main/views/test_users.py:250]:
```suggestion
```
Lets remove this?

File: app/main/helpers/diff_tools.py (hunk 2 above)
Comment (Revit) [reply app/main/helpers/diff_tools.py:5]: We already use zip_longest in line 102 so it should declare from itertools import zip_longest. No where in the file uses izip_longest

**Review by tobiasbrown — APPROVED** [review]

**Review by Revit — APPROVED** [review]

**Review by Reshma-M-A — COMMENTED** [review]
Looks good

**Pull request merged**